### ISSUE TITLE
[Grid Functionality] Customize MouseButton Clicks

### ISSUE THREAD

**Issue by Kithio** [raoulvdberge/refinedstorage#637]
After having started using this mod I noticed that Right Click drags out half a stack and Middle Click takes 1 item out... Which kind of annoys  me since I use right click to take 1 item from literally every single other inventory UI in the game!

So my proposition... Because I'm pretty sure you won't swap this around as a default...

Allow us to customize the input hotkeys for this in the in-game configs or just regular config file.

It's a pretty easy feature to create and I feel it should've been there to begin with so people like me who have an issue with mods changing from the defaults of UI integration can change it around for our own personal preference.

TL:DR; Allow us to change the button clicks for taking out items in the Grids

**Comment by raoulvdberge** [comment]
The game uses right click to take half a stack... or does MC have an option to change that?

**Comment by raoulvdberge** [comment]
I don't think MC gives the ability to configure mouse buttons for inventory interactions, so RS shouldn't too.
